resources/authgear/templates/en/web/authflowv2/settings.html

```diff
@@ -0,0 +1,322 @@
+        {{ if (and $any_name_readable $.FormattedName) }}<span class="truncate">{{ $.FormattedName }}</span>{{ end }}
+        {{ if (and (not $.FormattedName) $.EndUserAccountID) }}<span class="truncate">{{ $.EndUserAccountID }}</span>{{ end }}
+      </div>
+      <p class="body-text--lg">
+        {{ $editProfileHref := call $.MakeURL "/settings/profile"}}
+        {{ template "v2.page.settings.default.links.edit-profile" (dict "href" $editProfileHref )}}
+      </p>
+
+    </div>
+  </div>
+
+  {{/* Identity */}}
+  <div class="flex flex-col gap-2">
+    <div class="flex flex-col gap-1">
+      <div class="settings-title text-start">
+        {{ template "v2.page.settings.default.title-my-account" . }}
+      </div>
+      <div class="settings-description text-start">
+        {{ template "v2.page.settings.default.description-my-account" . }}
+      </div>
+    </div>
+
+    <div>
+    {{ range .IdentityCandidates }}
+      {{ if eq .type "login_id" }}
+      {{ if eq .login_id_type "email" }}
+      <a
+        class="settings-item w-full row-sep"
+        {{ if .identity_id }}
+          href="{{ call $.MakeURL "/settings/enter_login_id" "q_login_id_key" .login_id_key "q_login_id_type" .login_id_type "q_login_id_input_type" .login_id_input_type "q_identity_id" .identity_id }}"
+        {{ else }}
+          href="{{ call $.MakeURL "/settings/enter_login_id" "q_login_id_key" .login_id_key "q_login_id_type" .login_id_type "q_login_id_input_type" .login_id_input_type }}"
+        {{ end }}
+      >
+        <div class="flex w-full">
+          <div class="flex gap-2 w-44 items-center">
+            <i class="alternative-icon material-icons">
+              mail
+            </i>
+            {{ template "v2.page.settings.default.button-label-email" . }}
+          </div>
+          <div class="flex-1 flex-col text-start settings-description">
+            {{ range $i, $email := $.Emails }}
+            <div>
+              {{ $email }}
+            </div>
+            {{ end }}
+          </div>
+          <div class="flex items-center">
+            <i class="settings-item__alternative_icon material-icons">
+              arrow_forward_ios
+            </i>
+          </div>
+        </div>
+      </a>
+      {{ end }}
+
+      {{ if eq .login_id_type "phone" }}
+      <a
+        class="settings-item w-full row-sep"
+        {{ if .identity_id }}
+          href="{{ call $.MakeURL "/settings/enter_login_id" "q_login_id_key" .login_id_key "q_login_id_type" .login_id_type "q_login_id_input_type" .login_id_input_type "q_identity_id" .identity_id }}"
+        {{ else }}
+          href="{{ call $.MakeURL "/settings/enter_login_id" "q_login_id_key" .login_id_key "q_login_id_type" .login_id_type "q_login_id_input_type" .login_id_input_type }}"
+        {{ end }}
+      >
+        <div class="flex w-full">
```

Comment:
1. a block element is default full width. You don't need to set it to width 100%.
2. This div is necessary. Apply display flex to the anchor tag instead.